azure-eventgrid/src/main/java/com/microsoft/azure/eventgrid/implementation/TopicCredentials.java

```diff
@@ -1,4 +1,4 @@
-package com.microsoft.azure.eventgrid;
```

Comment:
Please move this class out of implementation package if you expect users to interact with it.